lib/std/segmented_list.zig

```diff
@@ -416,23 +516,23 @@ fn testSegmentedList(comptime prealloc: usize, allocator: *Allocator) !void {
     }
 
     testing.expect(list.pop().? == 100);
-    testing.expect(list.len == 99);
+    testing.expect(list.unmanaged.len == 99);
```

Comment:
perhaps there should be a `.length()` method?

Reply:
True. Maybe `len` so it's more consistent with the rest of the language.

I guess it also wouldn't hurt to add one to the unmanaged version.

Reply:
Never mind, I just realized there's a `count` method.